GitUI/UserControls/FileStatusList.Designer.cs

```diff
@@ -111,7 +111,7 @@ private void InitializeComponent()
             this.FilterWatermarkLabel.Name = "FilterWatermarkLabel";
             this.FilterWatermarkLabel.Size = new System.Drawing.Size(65, 13);
             this.FilterWatermarkLabel.TabIndex = 0;
-            this.FilterWatermarkLabel.Text = "Filter files...";
+            this.FilterWatermarkLabel.Text = "Filter files using a Regular Expression...";
```

Comment:
I would prefer regular expression here, GE (mostly) uses lowercase

Reply:
Changed. (First I thought to use upper case for the proper name "Regular Expression".)